actions/code-release/action.yml

```diff
@@ -5,137 +5,62 @@ inputs:
    token:
      description: "Token needed to access files "
      required: true
-   path-to-json:
-     description: "Path to json folder with version"
+   access-key:
+     description: "AWS access key"
      required: true
-   repository-name:
-     description: "Name of repository needed to create code packages"
+   secret-key:
+     description: "AWS secret key"
      required: true
-   repository-type:
-     description: "Repository type used in bash code"
+   version-number:
+     description: "version number"
      required: true
-   staging-bucket:
-     description: "Staging s3 bucket name"
+   lambda-name:
+     description: "lambda name"
      required: true
-   production-bucket:
-     description: "Production s3 bucket name"
+   s3-bucket:
+     description: "s3 bucket name"
      required: true
-   access-key-staging:
-     description: "AWS staging key"
+   zip-file-name:
+     description: "Name of the created zip file"
      required: true
-   secret-key-staging:
-     description: "AWS secret key staging"
+   txt-file-name:
+     description: "Name of the created txt file"
      required: true
-   access-key-production:
-     description: "AWS production key"
+   working-folder-name:
+     description: "Name of the folder in which the zip and txt files were created"
      required: true
-   secret-key-production:
-     description: "AWS production secret key"
-     required: true
-   source-directory:
-     description: "Name of repository needed to create code packages"
-     required: true
-   exclude-files:
-     description: "List of folders/files excluded in the release"
-     required: false
-   included-files:
-     description: "List of folders/files included in the release"
-     required: false
-   nested-folder:
-     description: "Folder name if files are nested"
-     required: false
    s3-layer-folder:
-     description: "S3 layers folder"
+     description: "S3 layers folder used in deploying layers"
      required: false
 
 runs:
    using: "composite"
    steps:
-      - name: Read verion number from json file
```

Comment:
Where do you want to get the version from ?

Reply:
the version number is an input variable that is passed from repo action

Reply:
Well I understand this however we were parsing the version here. Does it mean that you moved it out and every single repo will have to handle it itself? If so what was the reason?

Reply:
I moved version step to common action so in repositories we will have only steps for triggering the common one.